src/nni_manager/core/nniDataStore.ts

```diff
@@ -242,16 +242,10 @@ class NNIDataStore implements DataStore {
         const map: Map<string, MetricDataRecord[]> = new Map();
         const metrics: MetricDataRecord[] = await this.getMetricData(trialJobId, 'FINAL');
 
-        const multiPhase: boolean = await this.isMultiPhase();
-
         for (const metric of metrics) {
             const existMetrics: MetricDataRecord[] | undefined = map.get(metric.trialJobId);
             if (existMetrics !== undefined) {
-                if (!multiPhase) {
-                    this.log.error(`Found multiple FINAL results for trial job ${trialJobId}, metrics: ${JSON.stringify(metrics)}`);
-                } else {
-                    existMetrics.push(metric);
-                }
+                existMetrics.push(metric);
```

Comment:
Seems it need to contain the content from `if` condition, not `else`.

Reply:
The change is reverted, there is no change on nniDataStore now.